gatekeeper/src/main/java/eu/arrowhead/core/gatekeeper/service/GatekeeperDriver.java

```diff
@@ -241,8 +242,9 @@ public ICNProposalResponseDTO sendICNProposal(final Cloud targetCloud, final ICN
 		Assert.notNull(request, "Request is null.");
 		
 		final Relay relay = gatekeeperMatchmaker.doMatchmaking(new RelayMatchmakingParameters(targetCloud));
+		Session session = null;
```

Comment:
Without the finally block, there is no need to create session variable outside the try block.